packages/ERTP/src/typeGuards.js

```diff
@@ -0,0 +1,51 @@
+import { assert } from '@agoric/assert';
+import { Nat } from '@agoric/nat';
+
+/**
+ * Type guard for SetValue
+ *
+ * @param {Value} value
+ * @returns {value is SetValue}
+ */
+export const isSetValue = value => Array.isArray(value);
+
+/**
+ * Type guard for NatValue
+ *
+ * @param {Value} value
+ * @returns {value is NatValue}
+ */
+export const isNatValue = value => {
+  if (isSetValue(value)) {
+    return false;
+  }
+  return typeof Nat(value) === 'bigint';
+};
```

Comment:
Ah, I think there's a confusion about what the typeGuards are doing. They aren't meant to be validators, but rather to just tell Typescript which type we expect. The validation including asserts should be handled in other functions. I wasn't being consistent on this given the use of `Nat` which would throw, so I've changed it to `isNat`. 

Re: allowing `numbers` rather than `bigints`: I'm still allowing `numbers` in the code, but not in the JSDoc types, for backwards compatibility with strong encouragement to upgrade. Eventually I want to ensure that NatValues are all `bigints`, but not right now. The deadlines and other timer-related integers must all be `bigint` though, because they have direct comparisons without coercion which would error if not everything is a `bigint`.